### PR TITLE
US97 YP

### PR DESCRIPTION
**Pull request by YaroslavPetsenyuk**
Completed US97 Please review and approve.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
```
Comment (Alisherb28) [inline pom.xml:11]: I don't know.Is it problem when we have different version of idk?

**Review by AlisherT1987 — APPROVED** [review]
Looks good.Thank you

File: pom.xml [hunk 2]
```diff
@@ -9,8 +9,8 @@
...
-        <maven.compiler.source>18</maven.compiler.source>
-        <maven.compiler.target>18</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (AlisherT1987) [inline pom.xml]: I don't know.Is it problem when we have different version? Mine is 18

Comment (MFayazi17) [reply pom.xml]: Mine is 11. I think it is not going to support 18. Jilili, Vusal and I had this problem yesterday!

**Review by MFayazi17 — APPROVED** [review]
Approved and Merged!

**Review by SehribanC — APPROVED** [review]

**Review by MFayazi17 — APPROVED** [review]

**Review by SehribanC — APPROVED** [review]

**Pull request closed without merging**